fullstack-projects/auth-system/devashmit-day39/backend/auth.py

```
@@ -0,0 +1,11 @@
+from datetime import datetime,timedelta,timezone
+from jose import JWTError,jwt
+from passlib.context import CryptContext
+SECRET="change-me-long-random-secret";ALGO="HS256";EXP=60
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)